v201710/sandbox_test.go

```diff
@@ -933,3 +933,40 @@ func TestRateError(t *testing.T) {
 	wg.Wait()
 
 }
+
+func Testrequest(t *testing.T) {
```

Comment:
You should give this a better name that tells you what it's actually testing. What piece of functionality are you testing? What is the expected result?

Reply:
It is testing the request method in base.go. Expected result is error here because adServingOptimization is not set

Reply:
Yes, what I'm saying is that you should be more clear with the name of the test

Reply:
Also make sure it starts with TestSandbox like the rest of the tests in this file so when you run go test -v -run=Sandbox it'll run it